custom/icds_reports/management/commands/sql_scripts/update_ccs_data.sql

```diff
@@ -0,0 +1,36 @@
+DROP TABLE IF EXISTS temp_ccs_table;
+CREATE TABLE temp_ccs_table
+  AS (
+  SELECT
+    ccs.case_id as case_id,
+    ccs.month as month,
+    ccs.supervisor_id as supervisor_id,
+    case_list.complication_type as complication_type,
+    agg_bp.reason_no_ifa as reason_no_ifa,
+    agg_bp.new_ifa_tablets_total as new_ifa_tablets_total_bp,
+    agg_pnc.new_ifa_tablets_total as new_ifa_tablets_total_pnc,
+    agg_bp.ifa_last_seven_days as ifa_last_seven_days
+    FROM "ccs_record_monthly" ccs
+    LEFT OUTER JOIN "awc_location" awc ON ccs.awc_id = awc.doc_id AND awc.supervisor_id = ccs.supervisor_id
```

Comment:
can you directly use `ccs_record_monthly_view` instead of doing this location join here

Reply:
I tried that initially it gives this error ```ERROR:  complex joins are only supported when all distributed tables are joined on their distribution columns with equal operator```